### DISCUSSION TITLE
Docker Run Not Working

### DISCUSSION THREAD

**Discussion by wudc** [camptocamp/docker-mapserver#679] [Q&A]
Hi, I git checkout this repos and run the Docker run command. However, when I was reviewing the log, I saw this output:
wuda@wuda-1-2:~/Desktop/repos/mapserver/camptocamp/docker-mapserver$ sudo docker logs a273a716894a --follow --tail 100
Switching listen port to 80
[Mon Jan 01 16:52:29.663237 2024] [env:warn] [pid 1:tid 140078324942720] AH01506: PassEnv variable _ was undefined
[Mon Jan 01 16:52:29.667888 2024] [env:warn] [pid 1:tid 140078324942720] AH01506: PassEnv variable _ was undefined
 [notice] [pid 1] AH00489: Apache/2.4.52 (Ubuntu) mod_fcgid/2.3.9 configured -- resuming normal operations
 [notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'

What is the PassEnv value? I believe the start-server command is expecting it.
Also, I ran the Docker run command as a sudo user. Thus, I checked the localhost:80 and the MapServer is not working. I also tried localhost:8080. It didn't work either. 

My setup environment is VirtualBox 7.0 and Ubuntu 23.10 with Windows 10 as the host. Please help.

**Comment by sbrunner** [comment]
The pass PassEnv script will get all the environment variables and make them available for MapServer

What's your run command?

**Reply by radi2015** [reply]
docker run -d \
  --restart=unless-stopped \
  --volume=/Users/TU99MK0/Downloads/ms4w/apps/tutorial/:/mapdata/tutorial/:ro  ec41e5d6fefa

the same question
